rules/five-user-experiences-of-reporting-services/rule.md

```diff
@@ -1,39 +1,37 @@
 ---
 type: rule
 archivedreason: Only showing example screenshots
-title: Do you know the 5 user experiences of Reporting Services?
-guid: b66277bc-0091-402c-8d9b-ee3907eae909
+title: 'Do you know the 5 user experiences of Reporting Services?'
+guid: 67dd5881-cd15-4305-a3c9-aec4ca5d6e28
 uri: five-user-experiences-of-reporting-services
-created: 2023-12-11T11:38:33.0000000Z
-authors: 
-  - title: Jeoffrey Fischer
-    url: https://ssw.com.au/people/jeoffrey-fischer
-related:
-- customization-do-you-know-which-version-of-sql-reporting-services-and-visual-studio-you-are-using
-redirects: []
+created: 2013-07-02T07:51:10.0000000Z
+authors:
+  - title: Adam Cogan
+    url: https://ssw.com.au/people/adam-cogan
+  - title: Lei Xu
+    url: https://ssw.com.au/people/lei-xu
```

Comment:
Add yourself as well :)

Reply:
I am just migrating the rules, I don't think I should be an author